src/components/molecules/Visualizer/Engine/Cesium/hooks.ts

```diff
@@ -123,25 +135,41 @@ export default ({
     () => {
       if (initialCameraFlight.current) return;
       initialCameraFlight.current = true;
-      if (property?.default?.camera) {
+      if (property?.cameraLimiter?.cameraLimitterEnabled) {
+        engineAPI.flyTo(property?.cameraLimiter?.cameraLimitterTargetArea, { duration: 0 });
+      } else if (property?.default?.camera) {
         engineAPI.flyTo(property.default.camera, { duration: 0 });
       }
       const camera = getCamera(cesium?.current?.cesiumElement);
       if (camera) {
         onCameraChange?.(camera);
       }
     },
-    [engineAPI, onCameraChange, property?.default?.camera],
+    [
+      engineAPI,
+      onCameraChange,
+      property?.default?.camera,
+      property?.cameraLimiter?.cameraLimitterEnabled,
+    ],
     (prevDeps, nextDeps) =>
       prevDeps[0] === nextDeps[0] &&
       prevDeps[1] === nextDeps[1] &&
-      isEqual(prevDeps[2], nextDeps[2]),
+      isEqual(prevDeps[2], nextDeps[2]) &&
+      isEqual(prevDeps[3], nextDeps[3]),
```

Comment:
```suggestion
      prevDeps[3] === nextDeps[3],
```

Comparing bool can be done with `===`